test/ome-files/tiff.cpp

```diff
@@ -153,7 +164,7 @@ namespace
         idx[ome::files::DIM_CHANNEL] = idx[ome::files::DIM_MODULO_Z] =
         idx[ome::files::DIM_MODULO_T] = idx[ome::files::DIM_MODULO_C] = 0;
 
-      const char * const shades[] = {" ", "░", "▒", "▓", "█"};
+      const std::array<const char *,5> shades{{" ", "░", "▒", "▓", "█"}};
```

Comment:
The C++ code is 100% UTF-8 for certain.  Whether that's 100% portable is another matter though.